src/stores/Content.jsx

```diff
@@ -64,19 +65,18 @@ export default class ContentStore {
     // }
   }
 
-  stabilityFeeMarkdown = () => {
-    return (this.getHelpItem('stability-fee-information') || {}).markdown
-  }
+  stabilityFeeMarkdown = () => (this.getHelpItem('stability-fee-information') || {}).markdown
 
-  stabilityFeeContent = () => {
-    return this.stabilityFeeMarkdown() && compiler(this.stabilityFeeMarkdown());
-  }
+  stabilityFeeContent = () => this.stabilityFeeMarkdown() && compiler(this.stabilityFeeMarkdown());
 
-  showStabilityFeeAlert = () => {
-    if (this.stabilityFeeMarkdown() === '') {
-      return false;
-    }
-    return true;
+  showStabilityFeeAlert = () =>
```

Comment:
this is a boolean check no? `shouldShowStabilityFeeAlert`

Reply:
fixed